src/rcl_bindings.cpp

```diff
@@ -66,14 +66,14 @@ NAN_METHOD(CreateNode) {
     return;
   }
 
-  const char* node_name = *Nan::Utf8String(info[0]->ToString());
-  const char* name_space = *Nan::Utf8String(info[1]->ToString());
+  std::string node_name(*v8::String::Utf8Value(info[0]));
```

Comment:
Extract strings and save it in C++, otherwise rcl will report 'empty node name' error message in debug build, I guess the name string is still not right in release build, only no more check was performed.